### ISSUE TITLE
remove ONE word from this page: https://carto.com/developers/mobile-sdk/guides/pro-extensions/

### ISSUE THREAD

**Issue by agnesgyorfi** [CartoDB/mobile-sdk#332]
Hi, @CartoDB/rt-managers ! Would you please remove the word "military" from the page indicated immediately below? 🙇‍♀ 

https://carto.com/developers/mobile-sdk/guides/pro-extensions/

thanks!

**Comment by ztephm** [comment]
@csubira we used to open [`content` issues in Developer Center repo](https://github.com/CartoDB/developers/issues?q=is%3Aissue+is%3Aopen+label%3Acontent) for edits like this, but I think that's no longer monitored—assigning to Front-End for now but if it should go to someone else please let us know. I don't mind doing the pull request myself if we need(?)

**Comment by mtehver** [comment]
I just removed 'military' keyword from the guide in master branch of the SDK. I am not sure how to make the changes visible in CARTO webpage, though. It would be great if the API docs and guides could be updated to the most recent version (4.2.2), this has several new features compared to 4.1.3, which is shown on the webpage.

**Comment by csubira** [comment]
Thanks @mtehver! As you said, changes on `master` on the SDK repository,  are automatically fetched in Developers Center site. I triggered a new deploy to production so we don't have to wait until next scheduled deploy to have it live.
**I've checked and "military" is already removed!** 

Regarding the versions, I think we should tackle this issue in a new ticket where we specify two main factors: 
1. Do we want to keep 4.1.3 docs published in the web as well?
2. Are the new docs (guides and API Docs) for 4.2.2 already written?
Opened here to follow the discussion -> https://github.com/CartoDB/mobile-sdk/issues/333
